fedn/fedn/clients/reducer/restservice.py

```diff
@@ -211,6 +211,56 @@ def _scalar_metrics(metrics):
 
             return valid_metrics
 
+        @app.route('/map_view')
+        def map_view():
+            map = create_map()
+            try:
+                return render_template('index.html', show_map=True, map=map)
+            except Exception as e:
+                return str(e)
+
+        def create_map():
+            mapbox_access_token = "pk.eyJ1IjoiZHN0b3lhbm92YSIsImEiOiJja2cwbnBvYzUwMWFjMnhvNTY5azZ1NWs5In0.plMjsaU58g3jL9_50WSq9w"
```

Comment:
Is this OK to have in the code?

Reply:
It's a token for the Mapbox API which is linked to my account there. It's not a security risk when it comes to FEDn but whoever has the token, can use the Mapbox API without creating their own account. I can add it as a Git secret if you give me access to the Settings page of this repository.

Reply:
Then it will only work for any build job we setup, is there an alternative with a trial that would work for anyone starting this on their own machine? An api to request the token or something similar?

Reply:
If it is not possible to have the api working without a key I think we should look at alternatives like the openstreetmap/openlayers framework.

Reply:
I have now removed the Mapbox functionality and I have used `plotly.express` instead. No API access token is needed for it.